README.md

```diff
@@ -0,0 +1,39 @@
+This repository contains code for a privileged Firefox addon that measures
+rates of DNSSEC interference by network middleboxes. The entry point for the 
```

Comment:
Put a paragraph break between this line and the next part.

Reply:
Addressed by 41aa299fb054c1cacd06fd4c73d14adf7d1dfc68